src/components/header/header.styled.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+export const StyledHeader = styled.div`
```

Comment:
This is a different pattern than the one above for the App - align to a single pattern

Reply:
I'm going with ``export default ``